src/cli.rs

```diff
@@ -168,6 +170,17 @@ pub fn build_args<'a, 'b>(version: &'b str) -> App<'a, 'b> {
                 .required(false)
                 .help("If specified, transactions are send in batches of specified size"),
         )
+        .arg(
+            Arg::with_name("keeper-authority")
+                .long("keeper-authority")
```

Comment:
Am I right that you've renamed id into keeper-authority and reuse id now to specify id of a validator?